lib/oneview-sdk/resource/api1600/c7000/interconnect.rb

```diff
@@ -0,0 +1,37 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1200/c7000/interconnect'
+
+module OneviewSDK
+  module API1600
+    module C7000
+      # Interconnect resource implementation on API1600 C7000
+      class Interconnect < OneviewSDK::API1200::C7000::Interconnect
+        include OneviewSDK::ResourceHelper::ConfigurationOperation
+        # Create a resource object, associate it with a client, and set its properties.
+        # @param [OneviewSDK::Client] client The client object for the OneView appliance
+        # @param [Hash] params The options for this resource (key-value pairs)
+        # @param [Integer] api_ver The api version to use when interracting with this resource.
+        def initialize(client, params = {}, api_ver = nil)
+          @data ||= {}
+          # Default values
+          @data['type'] ||= 'InterconnectV7'
```

Comment:
There is no interconnectV7, please remove the initialize method if same method exist in API1200.

Reply:
done